src/js/stores/MesosFetchers.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import { MESOS_SUMMARY_CHANGE } from "#SRC/js/constants/EventTypes";
+import MesosSummaryStore from "./MesosSummaryStore";
+import Node from "../structs/Node";
+
+const getNodeFromSummary = (nodeId?: string, summary?: any) => {
+  const trimmedNodeID = nodeId
+    ? decodeURIComponent(nodeId).replace(/^\//, "")
```

Comment:
We have sentry bugs with decodeURIComponent breaking in certain cases. Do you think it makes sense to add some validation here?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Malformed_URI#Decoding